aviary/docs/user_guide/aviary_commands.ipynb

```diff
@@ -232,13 +232,14 @@
     "- TurboFan decks for both FLOPS and GASP can be converted\n",
     "- TurboProp decks for GASP can also be converted\n",
     "- Comments can be added using #\n",
+    "- The engine deck file that is to be converted must not have any headers\n",
```

Comment:
I'm requesting this line get removed.
GASP decks actually do have a "header" line with additional variables (not headers in the traditional sense because they don't label the data), so this line might actually cause confusion for GASP users. Also see my other comment on the PR.